contracts/freelancer_dispute_resolver.py

```
@@ -0,0 +1,158 @@
+        else:
+            raise Exception("Only the client or freelancer can submit evidence.")
+        self.dispute_raised = True
+
+    @gl.public.write
+    def resolve_dispute(self) -> None:
+        if not self.dispute_raised:
+            raise Exception("No dispute has been raised yet.")
+        if self.resolved:
+            raise Exception("Dispute has already been resolved.")
+        if not self.client_evidence:
+            raise Exception("Client has not submitted evidence yet.")
```

Ruff (0.15.1)

[warning] Create your own exception

(TRY002)